compiler/modules/nback.llvm/llvm.bal

```diff
@@ -169,6 +170,25 @@ public class Module {
         }
     }
 
+    // Corresponds to LLVMAddGlobal
+    public function addGlobal(Type ty, string name) {
```

Comment:
Should return PointerValue, like addFunctionDefn/Decl return a value.

C API returns a ValueRef.

Reply:
Fixed